chainercv/utils/iterator/apply_semantic_segmentation_link.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+
+from chainercv.utils.iterator.split_iterator import split_iterator
+
+
+def apply_semantic_segmentation_link(target, iterator, hook=None):
+    """Apply a semantic segmentation link to an iterator
+
+    This function applies a semantic segmentation link to an iterator.
+    It stacks the outputs of the semantic segmentation link
+    into :obj:`pred_labels`.
+    This function also stacks the values returned by the iterator.
```

Comment:
Not all values are stacked.
Perhaps, stacks the values returned by the iterator except images.